agent/config/types.go

```diff
@@ -143,6 +147,12 @@ type Config struct {
 
 	// Set if clients validate ssl certificates. Used mainly for testing
 	AcceptInsecureCert bool `json:"-"`
+
+	// MinSupportedCNIVersion denotes the minimum version of cni spec required
+	MinSupportedCNIVersion string
+
+	// CNIPluginsPath is the path for the cni plugins
+	CNIPluginsPath string
```

Comment:
yes, it's configurable from the environment variable `ECS_CNI_PLUGINS_PATH` as indicated in the readme file.